tests/protos/kitchen-sink.proto

```diff
@@ -7,6 +7,7 @@
 package some_package;
 
 import "tests/protos/options.proto";
+import "google/protobuf/descriptor.proto";
```

Comment:
I could but I would have to add support for "extensions" syntax in messages to specify what's reserved for extensions.